arcgis-ios-sdk-samples/Maps/Create and save a map/SaveAsViewController.swift

```diff
@@ -38,12 +49,80 @@ class SaveAsViewController: UITableViewController {
             presentAlert(message: "Please enter a title.")
             return
         }
-        
+        // Get the tags from the text field.
         let tags = tagsTextField.text?
             .components(separatedBy: ",")
             .map { $0.trimmingCharacters(in: .whitespacesAndNewlines) } ?? []
-        
+        // Get the item description from the text field.
         let itemDescription = descriptionTextField.text ?? ""
-        delegate?.saveAsViewController(self, didInitiateSaveWithTitle: title, tags: tags, itemDescription: itemDescription)
+        
+        delegate?.saveAsViewController(self, didInitiateSaveWithTitle: title, tags: tags, itemDescription: itemDescription, folder: selectedFolder)
+    }
+    
+    /// Toggles visisbility of the reference scale picker.
+    func toggleFolderPickerVisibility() {
```

Comment:
FYI: OSA Maps app uses an actionSheet for the folder picker

https://github.com/Esri/maps-app-ios/blob/d7ff3e3562351bf595f469c4921f5500a4abf593/maps-app-ios/UI/Account%20View/AccountDetailsViewController.swift#L87

But personally I like your picker here better because it makes the UI more consistent, so let's keep it.

Also I love the attention to the details below using `.accentColor` 👍

Reply:
As cool as inline pickers are, I'm not sure that it is the best choice for this UI. I say that because of the following from the [HIG](https://developer.apple.com/design/human-interface-guidelines/ios/controls/pickers/):

![image](https://user-images.githubusercontent.com/2257493/124314862-e230f980-db27-11eb-92ed-6813352f178a.png)

It might make more sense to display the folders in a separate table view controller that is pushed onto the navigation stack when the Folder row is tapped.